Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public class Player : MonoBehaviour, AbilityActor
```

Comment:
maybe a class Unit or Base instead of player*
that way we keep one monobehaviour